pdc/apps/package/models.py

```diff
@@ -27,6 +27,7 @@ class RPM(models.Model):
     release             = models.CharField(max_length=200, db_index=True)
     arch                = models.CharField(max_length=200, db_index=True)  # nosrc
     srpm_name           = models.CharField(max_length=200, db_index=True)  # package (name of srpm)
+    koji_build_target   = models.CharField(max_length=200, null=True, blank=True, db_index=True)
```

Comment:
As @sochotnicky explained in JIRA, it looks can't use built_for_release. If one koji build target is corresponding to one rpm and currently is not used by other models, it is much simplier just add a char field. 
Even move this field outside RPM model to some external module, is it still in PDC? If so, I don't quite understand why this follows the "PDC should be about metadata storage" rule.